### PR TITLE
osbuild: add support for `swap` in the bootupd stage

### PR DESCRIPTION
**Pull request by mvo5**
Trivial commit to allow swap in bootupd.

### CODE REVIEW

**Review by ondrejbudai — COMMENTED** [review]
Don't we need to handle swap on LV as well?

**Comment by mvo5** [comment]
> Don't we need to handle swap on LV as well?

Excellent point! Added now (my integration test run in bib did not quite yet go to the lvm part :)

**Review by ondrejbudai — APPROVED** [review]
Two nitpicks, but otherwise this is great! :)

File: pkg/osbuild/bootupd_stage.go [hunk 1]
```diff
@@ -110,17 +110,22 @@ func genMountsForBootupd(source string, pt *disk.PartitionTable) ([]Mount, error
...
+					mount, err := genOsbuildMount(lv.Name, payload)
+					if err != nil {
+						return nil, err
+					}
+					mount.Source = lv.Name
+					mounts = append(mounts, *mount)
+				case *disk.Swap:
+					// notthing to do
+				default:
 					return nil, fmt.Errorf("expected LV payload %+[1]v to be mountable, got %[1]T", lv.Payload)
```
[the remark this answers is not in the document]
Comment (mvo5) [reply pkg/osbuild/bootupd_stage.go]: Thanks, love both comments, both fixed, please re-review

**Review by ondrejbudai — APPROVED** [review]
Thanks! :)

**Pull request merged**